contracts/src/ecdsa/POAStakeRegistryStorage.sol

```diff
@@ -0,0 +1,45 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.27;
+
+import {CheckpointsUpgradeable} from
+    "@openzeppelin/contracts-upgradeable/utils/CheckpointsUpgradeable.sol";
+import {IPOAStakeRegistry} from "./interfaces/IPOAStakeRegistry.sol";
+
+/**
+ * @title Storage variables for the `POAStakeRegistry` contract.
+ * @author Layr Labs, Inc.
+ * @notice This storage contract is separate from the logic to simplify the upgrade process.
+ */
+abstract contract POAStakeRegistryStorage is IPOAStakeRegistry {
+    /// @notice The size of the current operator set
+    uint256 internal _totalOperators;
+
+    /// @notice Specifies the weight required to become an operator
+    uint256 internal _minimumWeight;
```

Comment:
confused by this, since it’s PoA there is no minimum. They are either whitelisted with a value or not. if not, 0. if whitelisted, some value >0

Maybe just leftover